### ISSUE TITLE
Adding path to plantUML image breaks confluenceUpload for Windows

### ISSUE THREAD

**Issue by Rosi2143** [docToolchain/docToolchain#582]
**Describe the bug**
When I run `gradle publishToConfluence` under Windows(10) I get this error on plantUML images:

```bash
3. Test levels
    image: images/W:/git/architecture/build/docs/images/plantUMLunit_test.png
allPages already retrieved
found existing page: 1565356910 version 43
page hasn't changed!

> Task :docToolchain:publishToConfluence FAILED

FAILURE: Build failed with an exception.

* Where:
Script 'W:\git\architecture\docToolchain\scripts\publishToConfluence.gradle' line: 27

* What went wrong:
Execution failed for task ':docToolchain:publishToConfluence'.
> java.io.FileNotFoundException: W:\git\architecture\build\docs\html5\images\W:\git\architecture\build\docs\images\plantUMLunit_test.png 
```

The path to the image - that does exist - is doubled.

It works when I run the same under Linux (Ubuntu 20.4)

**To Reproduce**
1. use this code

[ditaa, "{plantUMLDir}_unit_test", png]
----
     +-------------+
     |  Unit test  |
     +------+------+
            |
            v
  +-------------------+
  :Code under test    :
  |    +----+----+    |
  |    |   Unit  |    |
  |    +----+----+    |
  +-------------------+
            |
            v
     +------+-------+
     |     Stub     |
     +------+-------+
----
2. See error

When the path to {plantUMLDir} is removed it works.

variable :plantUMLDir: was tried as "" or "{imagesdir} == "images". But nothing changed.

**Expected behavior**
I hope I can set a path also here.

**Screenshots**
none.

**Configuration**
 - OS: Windows only - it works on Linux
 - Browser FireFox

**Additional context**
none

**Comment by rdmueller** [comment]
thanx for reporting this. I guess the problem is somewhere around this line:

https://github.com/docToolchain/docToolchain/blob/master/scripts/AsciiDocBasics.gradle#L107

I will investigate it tomorrow.

**Comment by rehsack** [comment]
Unfortunately the generated HTML files contain the FQPN to the generated images instead of relative paths, which needs to be fixed either. With this issue in mind, I submit a new patch soon which should fix both.

**Comment by Rosi2143** [comment]
Hello @rehsack,

anything available for testing yet?

**Comment by rdmueller** [comment]
Could it be that the FQPN was introduced with the PR https://github.com/docToolchain/docToolchain/pull/564 ?

The FQPN was introduced for PDF because it was hard to get the path for plantuml images right. For HTML, this wasn't a big problem and the plantUMLDir was set to "".

**Comment by rehsack** [comment]
No, the FQPN was already there and just have been moved from pure HTML targets to all targets.
